make/test.mk

```diff
@@ -180,7 +191,7 @@ deploy-host:
 ifeq ($(HOST_REPO_PATH),)
 	$(eval HOST_REPO_PATH = /tmp/codeready-toolchain/host-operator)
 endif
-	oc new-project $(HOST_NS) 1>/dev/null
+	-oc new-project $(HOST_NS) 1>/dev/null
```

Comment:
I am not sure why are we using `-` here and also to redirect error to `dev/null`,error code is `2` not `1`

Reply:
`-oc` with hyphen to ignore error and continue.  This is required as host-operator and registration-service will be deployed on same namespace `$(HOST_NS)` and both will try to create HOST_NS.

I check history and found that `1>/dev/null` was added to clean output with PR https://github.com/codeready-toolchain/toolchain-e2e/pull/13